Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,69 @@
 // 1.	Slide the header up when it is clicked on.
+$('header').on('click', function(){
+	$('header').slideUp(300);
+})
 
 // 2.	Remove the purplebox class
+$('.purplebox').removeClass('purplebox');
```

Comment:
Nice! You could have done this a number of ways, but this is my favorite: "does _anything_ have a class of purplebox? get rid of it!" :+1: